packages/corporate-integration/src/HypernetCorporateIntegration.ts

```diff
@@ -0,0 +1,144 @@
+import {
+	UUID,
+	AjaxError,
+	JsonWebToken,
+	EmailAddressString,
+	EthereumAccountAddress,
+} from "@hypernetlabs/objects";
+import { IAjaxUtils, AxiosAjaxUtils } from "@hypernetlabs/utils";
+import { apiBaseUrl } from "@corporate-integration/configuration";
+import {
+	IHypernetCorporateIntegration,
+	NFTMetadata,
+} from "@corporate-integration/IHypernetCorporateIntegration";
+
+import { injectable } from "inversify";
+import { okAsync, ResultAsync } from "neverthrow";
+import { Readable } from "stream";
+import jwt_decode, { JwtPayload } from "jwt-decode";
+
+@injectable()
```

Comment:
Just realized that this isn't really injectable. We can remove this decorator and then ditch inversify from package.json, to make this slightly smaller.